packages/website/src/types.d.ts

```diff
@@ -1,11 +1,19 @@
-import 'process';
+// TODO: remove after v2.0.0-beta.10
+declare module '*.svg' {
+  import type { ComponentType, SVGProps } from 'react';
+
+  const ReactComponent: ComponentType<SVGProps<SVGSVGElement>>;
+
+  // eslint-disable-next-line import/no-default-export
+  export default ReactComponent;
+}
```

Comment:
this can be removed if #4284 is merged

Reply:
do we want to remove this before merging?

Reply:
yes, I'm going to update this PR